data_hub/permissions.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+
+from rest_framework.permissions import BasePermission
+
+
+class CheckAPIKeyAuth(BasePermission):
+
+    # Compares if API key sent in request header is the same as defined
+    # secret API key
+    def has_permission(self, request, view):
+
+        if "API_KEY" in request.headers:
+            return request.headers["API_KEY"] == settings.API_SECRET_KEY
+
+        return False
```

Comment:
```suggestion
class CheckAPIKeyAuth(BasePermission):
    def has_permission(self, request, view):
        """
        Compares if API key sent in request header is the same as
        the defined secret API key.
        """

        return request.headers.get("API_KEY") == settings.API_SECRET_KEY
```
Doing it like this is more elegant.